### PR TITLE
Adding JSDoc and decorate README

### PR DESCRIPTION
**Pull request by JeanCarlosVR**
Adding best description and best references in readme about links, and JSDoc in ShardClient.ts

### CODE REVIEW

File: lib/ShardClient.ts [hunk 1]
```diff
@@ -113,6 +122,14 @@ export default class ShardClient extends EventManager {
     });
   }
 
+  /**
+   * Update Voice State
+   * [More](https://discord.com/developers/docs/resources/voice)
+   * @param data.channel_id Channel ID
```
Comment (Apacheli) [inline lib/ShardClient.ts:128]: Should be camelCase e.g. `channel_id` -> `channelID`

File: lib/ShardClient.ts [hunk 2]
```diff
@@ -134,7 +151,7 @@ export default class ShardClient extends EventManager {
   }
 
   /**
-   * Request members from a guild
+   * Request members from a guild [REST]
```
Comment (Apacheli) [inline lib/ShardClient.ts:154]: Not actually a REST method (but I will have REST implemented soon)

File: lib/ShardClient.ts [hunk 3]
```diff
@@ -103,6 +106,12 @@ export default class ShardClient extends EventManager {
 
   /**
    * Update presence
+   * [More](https://discord.com/developers/docs/game-sdk/activities)
+   * @param data Base
```
Comment (Apacheli) [inline lib/ShardClient.ts:110]: I prefer `@arg` over `@param`

**Review by Apacheli — CHANGES_REQUESTED** [review]

**Review by JeanCarlosVR — COMMENTED** [review]
Solved

**Pull request merged**